src/battle_util.c

```diff
@@ -2059,6 +2060,15 @@ u8 DoFieldEndTurnEffects(void)
             }
             gBattleStruct->turnCountersTracker++;
             break;
+        case ENDTURN_FOG: // Fog cannot be ended, unless Defog is used
```

Comment:
If we do add temp Fog, we need to do some changes here. :eyes:

Reply:
Oh you're right actually.

Reply:
Done.